docs/application/native/guides/device/system.md

```diff
@@ -197,6 +197,15 @@ The following table lists the download feature keys.
 
 The following table lists the FIDO feature keys.
 
+<a name="vibration"></a>
```

Comment:
Is it ok to add vibration key table here? You added `Vibration` as one of `System Keys` in above table.
However this table is added among `Feature Keys` list and even it is not following alphabetical order.

Reply:
I made modification.